tests/LayerTests.js

```diff
@@ -434,22 +431,18 @@ describe('cop', function () {
       });
 
     it('testHTMLLayerExample', function() {
-          makeObject1();
-        const htmlLayer = cop.basicCreate('LmakeHtmlLayer');
-        // FIXME: why are private functions used here?
-        // ensurePartialLayer, makeFunctionLayerAware
-        cop.ensurePartialLayer(htmlLayer, object1)["print"] =  function() {
-            return "<b>"+ cop.proceed() + "</b>";
-        };
-        htmlLayer.toString = function() {return "Layer HTML";};
-        cop.makeFunctionLayerAware(object1,"print");
+        makeObject1();
+        const htmlLayer = cop.create('LmakeHtmlLayer');
+        htmlLayer.refineObject(object1, {
+            print() { return '<b>' + cop.proceed() + '</b>' }
+        });
```

Comment:
I rewrote this test so it does not use the private functions ensurePartialLayer, basicCreate, make*LayerAware. I also changed a lot of other tests doing that.

@JensLincke, @onsetsu: was there a specific intent behind using those functions instead of testing the interface like it should be used by users?

I also removed testMakeFunctionLayerUnaware (at the bottom of this file's diff) because it only called makeFunctionLayerAware und makeFunctionLayerUnaware, but none of the uninstall or unrefine functions.